### DISCUSSION TITLE
essay/2021/09/27/%EA%B0%9C%EB%B0%9C%EC%9E%90%EA%B0%80-%EB%8F%88-%EB%B2%84%EB%8A%94-%EC%97%AC%EB%9F%AC-%EB%B0%A9%EB%B2%95%EB%93%A4

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#142] [Comments]
# essay/2021/09/27/%EA%B0%9C%EB%B0%9C%EC%9E%90%EA%B0%80-%EB%8F%88-%EB%B2%84%EB%8A%94-%EC%97%AC%EB%9F%AC-%EB%B0%A9%EB%B2%95%EB%93%A4

회사 취직 대기업 스타트업 SI 프리랜서 창업 여러명 창업 1인 개발 오픈소스 글쓰기 블로그에 광고 붙이기 기술 서적 집필 강의, 강연 오프라인 강의 대학교 강연 유튜브 온라인 강의(인프런, 패스트캠퍼스 등)

https://jeho.page/essay/2021/09/27/%EA%B0%9C%EB%B0%9C%EC%9E%90%EA%B0%80-%EB%8F%88-%EB%B2%84%EB%8A%94-%EC%97%AC%EB%9F%AC-%EB%B0%A9%EB%B2%95%EB%93%A4.html

**Comment by KyuseoShim** [comment]
프리랜서는 실력과 인맥이 있어야 성공할 수 있는 영역이라고 생각합니다. 깊게 얘기하면 한도 끝도 없고, 일반적으로 회사에서는 프리랜서(외주)는 그저 남으로 대하는 경우가 허다합니다. 필요할 때 함께 하지만, 필요 없어지면 가차 없는 경우가 있습니다. 장점은 다양한 비즈니스를 경험할 수 있고, 다양한 사람들을 만나고 기술을 경험해 볼 수 있습니다. 단점은 개발자의 경우 그 이상 이하도 기대하지 않으며 경력이 높을수록 비용대비 더 많은 일을 요구하게 됩니다. 사람처럼 대하는 분들을 만나는 것은 운이겠네요. 중니어 때 5년 정도의 경험으로 짧게 적어보려니 많은 얘기를 적을 수가 없네요. '운'이라는 요소와 개인의 성장을 얼마나 챙기느냐에 따라 프리랜서의 경험을 이로울수도 해로울수도 있을 것 같네요. 저의 경우에는 개인의 성장에 집중하지 못해 해로움이 더 많은 케이스였습니다. 이로움이 많은 경험을 가지신 분이 댓글을 남겨주시면 참고해 볼 수 있을 것 같네요 :D

**Reply by BenjaminKim** [reply]
비슷한 주제의 글이 [여기](https://jeho.page/essay/2021/08/24/%EA%B0%91%ED%98%95-%ED%94%84%EB%A6%AC%EB%9E%9C%EC%84%9C%EC%99%80-%EC%9D%84%ED%98%95-%ED%94%84%EB%A6%AC%EB%9E%9C%EC%84%9C.html)에 있습니다.

이 친구는 대부분의 프리랜서들과는 반대로 프리랜서 경험이 전혀 없다가 경력 마지막에 프리랜서를 하며 지내고 있네요.
